packages/sources/css/src/components/quantity/src/index.css

```diff
@@ -0,0 +1,108 @@
+@import '@vtmn/css-design-tokens/src/base-colors';
+@import '@vtmn/css-design-tokens/src/themes/default';
+@import '@vtmn/css-design-tokens/src/typography';
+@import '@vtmn/css-design-tokens/src/spacings';
+@import '@vtmn/css-design-tokens/src/opacities';
+
+.vtmn-quantity {
+  font: var(--vtmn-typo_font-family) var(--vtmn-typo_text-2-font-size)
+    var(--vtmn-typo_font-weight--normal);
+  color: var(--vtmn-semantic-color_content-primary);
+  height: fit-content;
+  width: auto;
+}
+
+.vtmn-quantity_container {
+  display: flex;
+  margin-block-start: var(--vtmn-spacing_1);
+  margin-block-end: var(--vtmn-spacing_1);
+}
+
+.vtmn-quantity input[type='number'] {
+  outline: 0;
+  width: rem(60px);
+  padding: 0 rem(2px);
+  background-color: var(--vtmn-semantic-color_background-primary);
+  box-shadow: inset 0 0 0 rem(2px) var(--vtmn-semantic-color_border-secondary);
+  text-align: center;
+  appearance: none;
+  z-index: 1;
+}
+
+.vtmn-quantity input[type='number']::placeholder {
+  color: var(--vtmn-semantic-color_content-tertiary);
+}
+
+.vtmn-quantity input[type='number']:not(:disabled):not(:focus):hover {
+  box-shadow: inset 0 0 0 rem(2px) var(--vtmn-semantic-color_border-secondary),
+    0 0 0 rem(3px) var(--vtmn-semantic-color_hover-primary);
+}
+
+.vtmn-quantity[aria-disabled='true'] {
+  opacity: var(--vtmn-opacity_disabled-state);
+  cursor: not-allowed;
+}
+
+/* Prevent the stack of opacity */
```

Comment:
What do you mean about "the stack of opacity"?

Reply:
.vtmn-btn already have an opacity set to 0,38 when disabled. So to prevent having to opacity stacking, I remove the opactity of disabled elements in this components.